src/components/Explore/NowPlaying.tsx

```diff
@@ -0,0 +1,50 @@
+import { css } from "@emotion/css";
+import React from "react";
+import { fetchLatestTracks } from "../../services/Api";
+import usePagination from "../../utils/usePagination";
+import { CenteredSpinner } from "../common/Spinner";
+import TrackList from "../common/TrackList";
+
+export const NowPlaying: React.FC = () => {
+  const [tracks, setTracks] = React.useState<Track[]>([]);
+
+  const { results, isLoading } = usePagination<Track>({
```

Comment:
Do we need the pagination hook here if we're not paginating the results?

Reply:
agree but it's on all other items on the home page so I made it the same the same. Will be back later